gobbler-server.rkt

```diff
@@ -222,9 +222,34 @@ waiting list.
 
 ;; GobblerUniverse iworld? -> GobblerBundle
 ;; Remove player from the game
-(define (drop-world uni world)
+(define (drop-world! uni world)
+  (set-game-queue! uni (drop-queued (game-queue uni) world))
+  (when (ready? uni)
+    (set-ready-players! uni (drop-player (ready-players uni) world)))
+  
   uni)
 
+;; [Listof iworld?] iworld? -> [Listof iworld?]
+;; Drop the iworld from the game queue
+(define (drop-queued queue world)
+  (drop-first (λ (other) (iworld=? world other))
+              queue))
+
+;; [Listof player?] iworld? -> [Listof player?]
+;; Drop the player with the iworld from the game
+(define (drop-player players world)
+  (drop-first (λ (player) (iworld=? world (player-iworld player)))
+              players))
+
+;; [A] (A -> Boolean) [Listof A] -> [Listof A]
+;; Drop the first match of pred in l
+(define (drop-first pred l)
```

Comment:
This is already defined for you https://docs.racket-lang.org/reference/pairs.html#%28def._%28%28lib._racket%2Fprivate%2Flist..rkt%29._remove%29%29

Reply:
I figured this already existed. Thanks!